src/pages/oauth/ClientAuthentication.njs

```diff
@@ -37,25 +45,53 @@ class ClientAuthentication extends Nullstack {
             <strong>E-mail</strong> { me.email }
           </p>
         </div>
+        <br />
```

Comment:
Best to apply margin in here instead of `<br>`. You can use `mb-4` in the above div